leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/QueueModeTest.java

```diff
@@ -0,0 +1,213 @@
+/*******************************************************************************
+ * Copyright (c) 2017 RISE SICS AB.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     RISE SICS AB - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.leshan.integration.tests;
+
+import static org.junit.Assert.assertArrayEquals;
+
+import org.eclipse.leshan.Link;
+import org.eclipse.leshan.core.request.ReadRequest;
+import org.eclipse.leshan.core.response.ReadResponse;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class QueueModeTest {
+
+    protected QueueModeIntegrationTestHelper queueModeHelper = new QueueModeIntegrationTestHelper();
+
+    @Before
+    public void start() {
+        queueModeHelper.initialize();
+        queueModeHelper.createServer((int) queueModeHelper.awaketime * 1000);
```

Comment:
I would store awaketime as attribute of this class and remove if from queueModelHelper